webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'kms',
-    filename: './kms.js',
+    library: 'KeyManagementService',
```

Comment:
Breaking change, but I think we're good here (since we're making a breaking change anyway).

Reply:
:) cool